Gemfile

```diff
@@ -1,59 +1,95 @@
 source 'https://rubygems.org'
+ruby '2.4.1'
 
-gem 'rails', '~> 4.2.10'
-ruby '2.3.7'
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
 
+gem 'rails', '5.0.6'
+gem 'puma', '~> 3.0'
```

Comment:
Just wondering, does this mean we can use threaded webserver now? Or is that a different `puma`?

Reply:
Yeah we can use threaded server @icarito